### ISSUE TITLE
Publish Pre-Compile to ES5

### ISSUE THREAD

**Issue by dleeyoda** [hildjj/node-cbor#60]
I was trying to `npm run build` in create-react-app and it failed to minify. create-react-app [recommends](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#npm-run-build-fails-to-minify) compiling into ES5. Could you publish pre-compiled to npm?

**Comment by klappy** [comment]
I am running into this issue as well. If anybody is able to confirm a solution or workaround, please post.

**Comment by crabicode** [comment]
Sadly, no further progress has been done :(

**Comment by crabicode** [comment]
@klappy 

Here is a dirty one-time script
```
npm i babel-cli -g && npm i --prefix ./node_modules/autobahn/node_modules/cbor --save-dev babel-preset-es2015 && /usr/local/bin/babel --presets es2015 ./node_modules/autobahn/node_modules/cbor/lib --out-dir ./node_modules/autobahn/node_modules/cbor/lib.es5 && mv ./node_modules/autobahn/node_modules/cbor/lib ./node_modules/autobahn/node_modules/cbor/lib.es6 && mv ./node_modules/autobahn/node_modules/cbor/lib.es5 ./node_modules/autobahn/node_modules/cbor/lib
```

**Comment by crabicode** [comment]
@klappy 

After digging an hour into the issue I found a quick fix

```
npm i --prefix ./node_modules/autobahn/node_modules/cbor babel-preset-env babel-polyfill --save-dev
/usr/local/bin/babel --presets env --plugins transform-runtime ./node_modules/autobahn/node_modules/cbor/lib --out-dir ./node_modules/autobahn/node_modules/cbor/lib.es5
mv ./node_modules/autobahn/node_modules/cbor/lib ./node_modules/autobahn/node_modules/cbor/lib.es6
mv ./node_modules/autobahn/node_modules/cbor/lib.es5 ./node_modules/autobahn/node_modules/cbor/lib
npm run build

**Comment by hildjj** [comment]
If someone sends me a PR that does the necessary things and doesn't impact the node build, i'll be happy to include it.

**Comment by hildjj** [comment]
`cbor-web` is now published at version 7.0.0, it's what you want.
